client/src/lib.rs

```diff
@@ -217,15 +214,13 @@ impl<V: vdaf::Client<16>, C: Clock> Client<V, C> {
 
     /// Shard a measurement, encrypt its shares, and construct a [`janus_core::message::Report`]
     /// to be uploaded.
-    fn prepare_report(&self, measurement: &V::Measurement) -> Result<Report, Error> {
+    fn prepare_report(&self, measurement: &V::Measurement, time: &Time) -> Result<Report, Error> {
```

Comment:
Could we take `std::time::Instant` here to further reduce this interface's dependency on Janus concepts? But maybe that's not a good idea because `Instant` isn't flexible enough.

Reply:
Instant wouldn't work, because that's often a monotonic counter and not wall clock time. We could take a `SystemTime` or a `chrono` type.

Reply:
SystemTime wouldn't require reexporting a particular version of chrono or time